src/main/resources/schema/type.graphqls

```diff
@@ -1,6 +1,141 @@
+type Vote{
+  id: ID!
+  targetID: ID!
+  voters: [User]
+  voteFor: Int
+  voteAgainst: Int
+}
+
+type Comment{
+  id: ID!
+  userID:ID!
+  message: String
+  votes: [Vote]
+  replies: [Reply]
+  createdAt: String
+  updatedAt: String
+}
+
+type Reply {
+  id:ID!
+  userID:ID!
+  message: String
+  votes: [Vote]
+  comments: [Comment]
+  createdAt: String
+  updatedAt: String
+}
+
 type Wish {
- id: ID!
- title: String!
- description: String!
- coverImageUrl: String
+  id: ID!
+  author: User
+  title: String!
+  description: String!
+  tags: String
+  coverImageUrl: String
+  votes: [Vote]
+  comments: [Comment]
+  createdAt: String
+  updatedAt: String
+}
+
+input WishInput{
+  author: ID
+  title: String!
+  description: String!
+  tags: String
+  coverImageUrl: String
+  votes: [ID]
+  comments: [ID]
+}
+
+
+interface User {
+  id: ID!
+  userName: String
+  password: String
+  userProfileImage: String
+  createdAt: String
+  updatedAt: String
+}
+
+enum DEV_TYPE{
+  FRONTEND
+  BACKEND
+  FULLSTACK
+}
+
+type Dev implements User{
+  id: ID!
+  userName: String
+  userEmail: String
+  password: String
+  userProfileImage: String
+  type: DEV_TYPE
+  tech: [String] #Languages,Frameworks, Tools, IDE's and anything you work with
+  stars: Int
+  xpLevel: Int
+  githubURL: String
+  twitterURL: String
+  createdAt: String
+  updatedAt: String
+}
+
+input DevInput {
+  userName: String
+  userEmail: String
+  password: String
+  userProfileImage: String
+  type: DEV_TYPE
+  tech: [String] #Languages,Frameworks, Tools, IDE's and anything you work with
+  stars: Int
+  xpLevel: Int
+  githubURL: String
+  twitterURL: String
+}
+
+type Wisher implements User{
+  id: ID!
+  userName: String
+  password: String
+  userProfileImage: String
+  wishList: [Wish]
+  feedTags: [String]
+  createdAt: String
+  updatedAt: String
+}
+
+type TeamMember{
```

Comment:
Again, teams are not a part of 'core functionality' that we planned - it will be done in second phase. Please, join Slack to keep updated :)